.github/pull_request_template.md

```diff
@@ -0,0 +1,10 @@
+We are no longer accepting new pull requrests in this repository
+================================================================
+
+Mbed Crypto is now maintained as part of Mbed TLS, see
+https://github.com/ARMmbed/mbedtls/issues/3064
+
+Please open your pull request in the mbedtls repository:
+https://github.com/ARMmbed/mbedtls/pulls
+
+Thanks for your interest in Mbed Crypto!
```

Comment:
What about to add that in issue_template.md" as well?

Reply:
I think it's there too, but for some reason the diff hides it at the end. (I'll never understand why we can't seem to come up with diff tools that generate more sensible diffs.)

Reply:
ok no issue then